src/main/jbake/content/download.adoc

```diff
@@ -83,9 +83,9 @@ IMPORTANT: Note: Information on possible ways to migrate from `javax` to `jakart
 |https://www.apache.org/dyn/closer.cgi/tomee/tomee-7.1.4/tomee-webapp-7.1.4.war[icon:download[] TomEE Webapp WAR] |7.1.4|25 Sep. 2020|29 MB |https://downloads.apache.org/tomee/tomee-7.1.4/tomee-webapp-7.1.4.war.asc[icon:download[] PGP] https://downloads.apache.org/tomee/tomee-7.1.4/tomee-webapp-7.1.4.war.sha512[icon:download[] SHA512]
 |===
 
-IMPORTANT: Note: TomEE 7.1.4 was released two years ago. New releases from this branch are **unlikely**, but we are open for contributors to step up and lead the efforts in fixing CVEs and updating dependencies.
+IMPORTANT: This branch is discontinued. No upcoming releases are planned due to transitive dependencies reached their end of life.
```

Comment:
Maybe:

> This branch is discontinued. No upcoming releases are planned due to transitive dependencies reached their end of life, i.e. they might be affected by security vulnerabilities.


Just to make it clear?

Reply:
Agree. Thank you for the feedback @rzo1 . I'm pushing now the improved text.